concepts/05 UI Components/Diagram/05 Overview/05 Elements.md

```diff
@@ -0,0 +1,10 @@
+<img src="/images/diagram/diagram-elements.png" alt="Diagram elements"/>
+
+1. [Toolbar](/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/mainToolbar/)
+2. [Toolbox](/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/toolbox/)
+3. [View Toolbar](/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/viewToolbar/)
+4. [Diagram Page](/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/pageSize/)
```

Comment:
We should update or decide if we want to keep it at all - https://js.devexpress.com/React/Documentation/Guide/UI_Components/Diagram/Diagram_Tools/

Please make a decision about this page